kitsune/community/static/js/ContributorsList.jsx

```diff
@@ -0,0 +1,28 @@
+import {Paginator} from './contributors-common.jsx';
```

Comment:
Oops, I had called it foo originally, but renamed it to be clearer (ironically, making it more confusing). It should actually be:

```js
import _contributors_common from './contributors-common.jsx';
const Paginator = _contributors_common.Paginator;
```